Rakefile

```diff
@@ -72,7 +84,10 @@ namespace :test do
   Rake::TestTask.new('all') do |t|
     t.libs << BASE_DIR
     t.test_files = FileList["#{BASE_DIR}/core/test_*.rb",
-                            "#{BASE_DIR}/plugins/multi-nodes/test_*.rb"]
+                            "#{BASE_DIR}/multi-nodes/test_*.rb",
```

Comment:
Actually, let's remove the multi-nodes tests from that list, as they require a very specific setup (multiple Kill Bill instances). We rarely run them (and always manually).